git-guide/git-topic-1/index.html

```diff
@@ -56,124 +56,224 @@
-      </a>
     </div>
   </div>
+</div>
 
-    </div>
+<div class="ui segment pushable" >
 
-    <div class="three wide column">
+  <div class="ui inverted labeled icon left inline vertical sidebar menu">
+    
+  </div>
+  <div class="pusher" tabindex="-1">
+    <div class="ui basic segment" id="labchat">
+      <br>
       
-    </div>
+      <div  class="ui tab segment lab" data-tab="Lab-01-Branching and Merging">
+        <h1>Objectives</h1>
+          <ol>
+            <li>Understand what A Terminal is</li>
+            <li>Understand Basic Terminal Commands</li>
+          </ol>
 
-
-  </header>
-  
-  
-   <div class="ui segment pushable">
-    <div class="ui inverted labeled icon left inline vertical sidebar menu">
-      <div class="header">
-        <a class="item" href="#">
-          
-        </a>
       </div>
-      <div class="menu">
-        
-          <a class="item" href="talk-1-git/git.pdf">
-            Git
-          </a>
-        
-          <a class="item" href="book-b-branching-and-merging/index.html">
-            Lab-01-Branching and Merging
-          </a>
-        
+     
+      <div  class="ui tab segment lab" data-tab="01">
+        <h1>Topics</h1>
+<p>In this workshop, we&#39;ll cover the following:</p>
+<h2>Terminal</h2>
+<p>Basi Terminal Commands</p>
+
       </div>
-    </div>
-    <div class="pusher">
-      <div class="ui basic container segment">
-       
-         
-  
+     
+      <div  class="ui tab segment lab" data-tab="02">
+        <h1>Terminal</h1>
+<h2>What is a terminal?</h2>
+<p>The terminal is an interface in which you can type and execute text based commands.</p>
 
-         
-         
-  
-    <section class="ui attached segment">
-      <a class="ui right ribbon label">talks</a>
-      <div class="ui centered four doubling stackable cards">
-  
-     <a class="ui raised link card" href="talk-1-git/git.pdf" target="_blank">
-  
+<p float="left"><img src="img/terminal.png" alt="A Terminal View"></p>
 
-      <div class="center aligned content">
-        <div class="right floated meta"><i class="huge object group icon"></i></div>
-        <div class="header"> Git</div>
-      </div>
+<p>Another way of describing an interface is how something is interacted with</p>
+<p>Some examples are<p>
+<ul>
+<li>A Steering wheel in a car</li>
+<li>A Control on a Games Console</li>
+<li>A Keyboard on a computer</li>
+<li>GUI - Graphical User Interface</li>
+</ul>
 
-      <div class="image">
-        <img class="ui bordered rounded image" src="talk-1-git/git.png">
-      </div>
+<p>Can you think of any other examples?</p>
+
+<h2>Alternative Names for Terminal</h2>
+  <ul>
+    <li>Console</li>
+    <li>Command Line</li>
+    <li>Command Prompt</li>
+    <li>Shell</li>
+  </ul>
+
+<h2>How to get access to your terminal</h2>
+
+<p> This depends on the operating system you are using</p>
+<ul>
+<li>On Mac - use the SearchLight Function</li>
+<li>On Linux - use the Search Program Function</li>
```

Comment:
Might be worth adding windows here too? If they have access to a computer at home, it will most likely be windows